examples/99-advanced/planets.py

```diff
@@ -0,0 +1,119 @@
+"""
+.. _planets_example:
+
+3D Earth and Celestial Bodies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Creates high-resolution renderings of the Earth and the major celestial bodies
+in our solar system for astrodynamics applications.
+This example is inspired by `planet3D-MATLAB <https://github.com/tamaskis/planet3D-MATLAB>`_ .
+
+
+Note that this example author doesn't know anything about astronomy. There may
+be inaccuracies in the representation. The purpose of this example is for
+inspiration only.
+
+"""
+import pyvista
+from pyvista import examples
+
+###############################################################################
+# Celestial bodies with the Milky Way in the background.
```

Comment:
I don't actually see the Milky Way in the result. Should we rotate the whole scene around to get some of the Milky Way in the frame?

Reply:
Oh. I changed it. Thanks.

Reply:
```suggestion
# Celestial bodies with the stars in the background.
```